Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM alpine
+FROM golang:alpine3.13 AS builder
+LABEL stage=builder
```

Comment:
Is that label necessary?

Reply:
Well no, it is a label. I wanted to make clear that this step is only needed for building the software. But I can delete it if you want.